test/apps/test_fuzzy_match.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python2
+
+import os
+import commands
+import re
+import argparse
+import sys
+import logging
+
+###################
+# Main test runner
+###################
+class FuzzyTester:
+    def __init__(self, ucx_info, verbose):
+        self.ucx_info = ucx_info
+        if verbose:
+            logging.basicConfig(level=logging.DEBUG)
+        
+    def cleanup(self):
+        ucx_vars = [var for var in os.environ.keys() if var.startswith('UCX_')]
+        for var in ucx_vars:
+            del os.environ[var]
+
+    def run(self, env_vars):
+        self.cleanup()
```

Comment:
Typical case for using a [context manager](https://stackoverflow.com/questions/1984325/explaining-pythons-enter-and-exit) instead of this.

Reply:
See also
https://stackoverflow.com/questions/2059482/python-temporarily-modify-the-current-processs-environment